web/src/api/open-api/api.ts

```diff
@@ -1344,6 +1344,23 @@ export interface SmartInfoResponseDto {
      */
     'objects'?: Array<string> | null;
 }
+/**
+ * 
+ * @export
+ * @enum {string}
+ */
+
+export const SystemConfigKey = {
+    Crf: 'ffmpeg_crf',
+    Preset: 'ffmpeg_preset',
+    TargetVideoCodec: 'ffmpeg_target_video_codec',
+    TargetAudioCodec: 'ffmpeg_target_audio_codec',
+    TargetScaling: 'ffmpeg_target_scaling'
+} as const;
```

Comment:
The keys in this generated enum don't have the ffmpeg prefix, do you have any idea why?

Reply:
I'm not sure about that.